ChangeLog

```diff
@@ -0,0 +1,24 @@
+Fri Mar 29 2019 Piotr Balcer <piotr.balcer@intel.com>
+
+	* Version 0.8
+	This is the first official release of a libvmemcache. It's an embeddable
+	and lightweight in-memory caching solution designed to fully take
+	advantage of large capacity memory, such as Persistent Memory with DAX,
+	through Memory Mapping in an efficient and scalable way.
+
+	It includes, among other things:
+	- Extent-based memory allocator which sidesteps the
```

Comment:
(editing style) please move 'the' from the end to the beginning of the next line